addons/payment/wizards/payment_link_wizard.py

```diff
@@ -62,17 +63,60 @@ def _compute_values(self):
     @api.depends('res_model', 'res_id')
     def _compute_company(self):
         for link in self:
-            record = self.env[link.res_model].browse(link.res_id)
-            link.company_id = record.company_id if 'company_id' in record else False
+            related_document = self.env[link.res_model].browse(link.res_id)
+            link.company_id = related_document.company_id
+
+    @api.depends('payment_acquirer')
+    def _compute_has_multiple_acquirer(self):
+        for link in self:
+            if len(self._get_payment_acquirer_available()) > 2:
+                link.has_multiple_acquirer = True
+            else:
+                link.has_multiple_acquirer = False
+
+    def _get_payment_acquirer_available(self):
+        """ Specify available acquirers in the selection field.
+
+        :return: a list of tuples, the first one being the 'all' one, and then
+        one tuple for each available acquirer, with state enabled or test.
```

Comment:
This is overly specific IMO. This could be shortened to something like `:return: The selection list of available acquirers`.